vcd/config_test.go

```diff
@@ -228,7 +228,7 @@ provider "vcd" {
   vdc                  = "{{.Vdc}}"
   allow_unverified_ssl = "{{.AllowInsecure}}"
   max_retry_timeout    = {{.MaxRetryTimeout}}
-  version              = "~> {{.VersionRequired}}"
+  #version             = "~> {{.VersionRequired}}"
```

Comment:
Do we need to comment the comment?

Reply:
I disabled the line as a directive, but I've left there as a comment to help debugging.